pkg/providers/instancetype/nodeclass_test.go

```diff
@@ -645,16 +646,8 @@ var _ = Describe("NodeClass/InstanceTypes", func() {
 	Context("Overhead", func() {
 		var info *ec2.InstanceTypeInfo
 		BeforeEach(func() {
-			ctx, err := (&settings.Settings{}).Inject(ctx, &v1.ConfigMap{
-				Data: map[string]string{
-					"aws.clusterName": "karpenter-cluster",
-				},
-			})
-			Expect(err).To(BeNil())
-
-			s := settings.FromContext(ctx)
-			ctx = settings.ToContext(ctx, test.Settings(test.SettingOptions{
-				VMMemoryOverheadPercent: &s.VMMemoryOverheadPercent,
+			ctx = options.ToContext(ctx, test.Options(test.OptionsFields{
+				ClusterName: lo.ToPtr("CLUSTER_NAME"),
```

Comment:
Just curious: any specific reason to change this value?

Reply:
Miss on my part, idk if it matters but I'll change it back to karpenter-cluster.